sdk/nodejs/runtime/invoke.ts

```diff
@@ -78,6 +81,18 @@ export function invoke(tok: string, props: Inputs, opts: InvokeOptions = {}): Pr
     return invokeSync(tok, props, opts, syncInvokes);
 }
 
+export function streamInvoke(
+    tok: string,
+    props: Inputs,
+    opts: InvokeOptions = {},
+): AsyncIterable<any> {
+    if (opts.async) {
+        throw Error("streamInvoke does not support async mode");
```

Comment:
Seems like `streamInvoke` _only_ supports async mode?

Reply:
Yes! On purpose. By definition it's an `AsyncIterable`.

Reply:
Perfect. I don't think we need to throw in this case, then--`opts.async` is essentially meaningless, so we might as well ignore it entirely.

Reply:
I still think we shouldn't throw here.

Reply:
oops missed this. will fix tonight.

Reply:
Ok I just made `streamInvoke` async natively.